src/main/java/com/answerdigital/benhession/academy/answerkingweek2/services/OrderService.java

```diff
@@ -19,15 +22,17 @@ public class OrderService {
 
     private final ItemService itemService;
 
+    private final OrderMapper orderMapper =
+            Mappers.getMapper(OrderMapper.class);
+
     @Autowired
     public OrderService(final OrderRepository orderRepository, final ItemService itemService) {
         this.orderRepository = orderRepository;
         this.itemService = itemService;
     }
 
-    public Order addOrder(final String address) {
-        final Order order = new Order(address);
-        return orderRepository.save(order);
+    public Order addOrder(final OrderRequest orderRequest) {
+        return orderRepository.save(orderMapper.addRequestToOrder(orderRequest));
```

Comment:
It might be just my preference, but I think separating functions would help for readability.
In this case write it in two separate lines like below, instead of nesting functions. 
I've seen it in many places in this commit, so don't expect to change, but just a consideration. 
```
Order newOrder = orderMapper.addRequestToOrder(orderRequest);
        return orderRepository.save(newOrder);
```

Reply:
I think the same, it would be much more readable if we clearly see the mapping in a separate line.

Reply:
Agreed! Will fix throughout!

Reply:
Fixed